framework/src/main/java/org/checkerframework/framework/stub/RemoveAnnotationsForInference.java

```diff
@@ -76,6 +103,28 @@ public static void main(String[] args) {
       System.err.println("Usage: provide one or more directory names to process");
       System.exit(1);
     }
+    // TODO: using plume-lib's options here would be better, but would add a dependency
+    // to the whole Checker Framework, which is undesirable. Move this program elsewhere
+    // (e.g., to a plume-lib project)?
+    if (args[0].contentEquals("-exclusionFile")) {
+      if (args.length < 3) {
```

Comment:
I would have tested for length < 2 here, and then on line 118 set args and ended the `if "-exclusionFile"`.  Then move the `if args.length < 1` test to just after the end of the `if "-exclusionFile"` then clause.  I think that satisfies the Index Checker and does each test just once.  But I you prefer it this way, that's OK.

Reply:
Ah, I understand what you mean! Sorry for the confusion before - I misunderstood your suggestion. I agree that that is simpler, so I've implemented your suggestion.